modules/surrealdb/examples_test.go

```diff
@@ -0,0 +1,37 @@
+package surrealdb_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/surrealdb"
+)
+
+func ExampleRunContainer() {
+	// runSurrealDBContainer {
+	ctx := context.Background()
+
+	surrealdbContainer, err := surrealdb.RunContainer(ctx, testcontainers.WithImage("surrealdb/surrealdb:v1.1.1"))
+	if err != nil {
+		panic(err)
+	}
+
+	// Clean up the container
+	defer func() {
+		if err := surrealdbContainer.Terminate(ctx); err != nil {
+			panic(err)
```

Comment:
In #2193 we are replacing panics with log.Fatalf. Could you proceed as in that PR with this module? I'll probably merge that PR first, as it replaces all the panics in testable examples with log.Fatal

Reply:
Done!